dev-site-config/site.config.js

```diff
@@ -14,6 +14,8 @@ const siteConfig = {
       'Default Theme': '',
       'Legacy Theme': 'terra-legacy-theme',
     },
+
+    locales: ['ar', 'en', 'en-US', 'en-GB', 'es', 'es-US', 'es-ES', 'de', 'fi-FI', 'fr', 'fr-FR', 'nl', 'nl-BE', 'pt', 'pt-BR', 'sv', 'sv-SE'],
```

Comment:
Are we able to remove this?

Reply:
Yeah, we can remove this. It will then start using the default locales we have in toolkit which will be updated to include sv and sv-SE later today when we cut the release.